src/main/java/com/eerussianguy/blazemap/profiling/Profilers.java

```diff
@@ -42,15 +42,19 @@ public static class Mixin {
     }
 
     public static class Minimap {
-        public static final TimeProfilerSync DRAW_TIME_PROFILER = new TimeProfilerSync("draw", 60);
+        public static final TimeProfilerSync DRAW_TIME_PROFILER = new TimeProfilerSync("draw", 200);
         public static final TimeProfilerSync TEXTURE_TIME_PROFILER = new TimeProfilerSync("texture", 60);
         public static final LoadProfiler TEXTURE_LOAD_PROFILER = new LoadProfiler(60, 16);
     }
 
     public static class FileOps {
         public static final TimeProfilerAsync CACHE_READ_TIME_PROFILER = new TimeProfilerAsync("cache_read", 20);
+        public static final LoadProfiler CACHE_READ_LOAD_PROFILER = new LoadProfiler(20, 50);
         public static final TimeProfilerAsync CACHE_WRITE_TIME_PROFILER = new TimeProfilerAsync("cache_write", 20);
+        public static final LoadProfiler CACHE_WRITE_LOAD_PROFILER = new LoadProfiler(20, 50);
         public static final TimeProfilerAsync LAYER_READ_TIME_PROFILER = new TimeProfilerAsync("layer_read", 20);
+        public static final LoadProfiler LAYER_READ_LOAD_PROFILER = new LoadProfiler(20, 50);
         public static final TimeProfilerAsync LAYER_WRITE_TIME_PROFILER = new TimeProfilerAsync("layer_write", 20);
+        public static final LoadProfiler LAYER_WRITE_LOAD_PROFILER = new LoadProfiler(20, 50);
```

Comment:
I have an issue here that I think I raised before, and if I didn't, I should have:

Some of these file ops can be either client or server, because they belong to parts of the engine that _may_ replicate on both sides. So they should be in the Client or Server container classes.

One example is the cache, which could be being used on both sides, even though the server skips that step.
The way a reusable component knows what profilers to use is simple: pass in constructor, parent in theory already has the right one. See, for example, how it is done in the Pipeline class.

This is not critical and we can make a build as is... but sooner or later this should be rectified before we forget it.